rand_chacha/Cargo.toml

```diff
@@ -15,10 +15,10 @@ categories = ["algorithms", "no-std"]
 edition = "2018"
 
 [dependencies]
-rand_core = { path = "../rand_core", version = "0.6.0" }
+rand_core = { path = "../rand_core", version = "0.6.0", default-features = false }
 ppv-lite86 = { version = "0.2.8", default-features = false, features = ["simd"] }
 
 [features]
-default = ["std"]
```

Comment:
`rand_core/std` enables `rand_core/getrandom`, so I don't think we need to have `rand_core/getrandom` listed separately here?

Reply:
Thanks, I find it a bit surprising though. Extracted `getrandom` in a separate feature now in 19de8ba.
Let me know if there is anything else that needs to be changed.